src/main/java/org/entando/plugins/pda/core/model/Task.java

```diff
@@ -1,20 +1,29 @@
 package org.entando.plugins.pda.core.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import java.util.Map;
-import lombok.Builder;
-import lombok.Data;
-
-@Data
-@Builder
-public class Task {
-    private String id;
-    private String name;
-    private String subject;
-    private String description;
-    private String status;
-    private Integer priority;
-    private Boolean skipable;
-    private String processId;
-    private String processInstanceId;
-    private Map<String,String> properties;
+import lombok.NoArgsConstructor;
+import org.entando.web.response.BaseMapModel;
+
+@NoArgsConstructor
+public abstract class Task extends BaseMapModel {
+
+    public Task(Map<String,Object> map) {
+        super(map);
+    }
+
+    @JsonIgnore
```

Comment:
Are these ignored because they'll also show up in the Map<>?

Reply:
Yes, they would show up duplicate, one with the method name and another with the map property name.

Reply:
👍